### PR TITLE
Bugfix/804 788 787 779 778

### PR DESCRIPTION
**Pull request by MishShevtsov**
Complete collection of bugfix: 779 778 787 788 804

### CODE REVIEW

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss [hunk 1]
```diff
@@ -18,11 +27,24 @@ hr {
...
-    margin: 0 32px 40px 0;
+.list {
+  list-style: none;
+  padding-left: 0;
 }
 
-.list {
-    list-style: none;
-    padding-left: 0;
+// .gallery-view-li-active {
```
Comment (RomanDeveloperAcc) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss:35]: General: please, remove commented code

**Comment by mplesha** [comment]
conflicts

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 2]
```diff
@@ -6,7 +6,7 @@ div {
 
 .list-gallery {
   margin-bottom: 32px;
-  width: 359px;
+  // width: 359px;
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:9]: commented code

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss [hunk 3]
```diff
@@ -99,13 +98,13 @@ div {
 
 @media screen and (max-width: 1200px) {
   .list-gallery {
-    width: 299px;
+    // width: 299px;
```
Comment (mplesha) [inline src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss:101]: commented code

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss [hunk 4]
```diff
@@ -162,3 +169,38 @@ $main-font: 'Open Sans';
...
+            margin-left: 0px;
+        }
+    }
+}
+
+@media only screen and (max-width: $bp-smallest) {
+    .news-info {
+        flex-direction: column;
+    }
+}
```
Comment (KimAT25) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss:206]: add new line in the end

**Comment by mplesha** [comment]
conflicts

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss (hunk 1 above)
Comment (MishShevtsov) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.scss]: Done

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss (hunk 2 above)
Comment (MishShevtsov) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss]: Done

File: src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss (hunk 3 above)
Comment (MishShevtsov) [reply src/app/component/eco-news/components/news-list/news-list-gallery-view/news-list-gallery-view.component.scss]: Done

File: src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss (hunk 4 above)
Comment (MishShevtsov) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-detail.component.scss:206]: Done

**Pull request merged**